src/promptflow/promptflow/_sdk/_service/apis/collector.py

```diff
@@ -28,7 +29,9 @@
 from promptflow._utils.thread_utils import ThreadWithContextVars
 
 
-def trace_collector(logger: logging.Logger):
+# Pass in the get_created_by_info_with_cache and logger to avoid app related dependency.
```

Comment:
Nit. consider make this comment with docstring, instead of comments, so that we can see these when we write code calling this function.